### PR TITLE
core(audits): mention <div> is allowed to group content in <dl>

### PR DESCRIPTION
**Pull request by Malvoz**
Adds a mention that `<div>` is allowed to group content in `<dl>`, reference:  https://github.com/dequelabs/axe-core/pull/1076.

(Equivalent PR for updating the same thing in the axe repo: https://github.com/dequelabs/axe-core/pull/2098.)

### CODE REVIEW

File: lighthouse-core/audits/accessibility/definition-list.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ const i18n = require('../../lib/i18n/i18n.js');
 const UIStrings = {
   /** Title of an accesibility audit that evaluates if all the definition list elements have valid markup for screen readers. This title is descriptive of the successful state and is shown to users when no user action is required. */
   title: '`<dl>`\'s contain only properly-ordered `<dt>` and `<dd>` groups, `<script>` ' +
-      'or `<template>` elements.',
+      ', `<template>` or `<div>` elements.',
```
Comment (patrickhulce) [inline lighthouse-core/audits/accessibility/definition-list.js]:
```suggestion
      '`<template>` or `<div>` elements.',
```

File: lighthouse-core/audits/accessibility/definition-list.js [hunk 2]
```diff
@@ -16,10 +16,10 @@ const i18n = require('../../lib/i18n/i18n.js');
 const UIStrings = {
   /** Title of an accesibility audit that evaluates if all the definition list elements have valid markup for screen readers. This title is descriptive of the successful state and is shown to users when no user action is required. */
   title: '`<dl>`\'s contain only properly-ordered `<dt>` and `<dd>` groups, `<script>` ' +
```
Comment (patrickhulce) [inline lighthouse-core/audits/accessibility/definition-list.js]:
```suggestion
  title: '`<dl>`\'s contain only properly-ordered `<dt>` and `<dd>` groups, `<script>`, ' +
```

Comment (patrickhulce) [reply lighthouse-core/audits/accessibility/definition-list.js]: we've got a space between the comma at the moment :)

**Comment by Malvoz** [comment]
@patrickhulce This is passing now, sorry for a messy PR! 😅

**Pull request merged**